gitpod-io/gitpod
# [GHA] add VS Code Browser release

## Description


## Related Issue(s)

Fixes ENT-64

## How to test

- Smoke test VS Code Browser with versions are all working well
- it should generate no changes with cmd below
```
cd /workspace/gitpod/components/ide/code/gha-update-image/
yarn
bun run index.ts
```

|Editor|Stable Code| Pinned code (index 0)|Pinned code (index -1)| latest Code |
|:-:|:-:|:-:|:-:|:-:|
|✅ |<img width="1166" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/3a2b235a-b700-4544-94a6-4ff4c3e92c56">|<img width="1182" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/d97ca29d-3eed-4e10-8448-e1a04c60363a">|<img width="1169" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/31b5955b-5d81-466d-935a-582c90270fd3">|No worth test it as it's what configured with WORKSPACE.yaml|




## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - hw-code-gha</li>
	<li><b>🔗 URL</b> - <a href="https://hw-code-gha.preview.gitpod-dev.com/workspaces" target="_blank">hw-code-gha.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - hw-code-gha-gha.25218</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-hw-code-gha%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-dev-preview" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [x] with-integration-tests=vscode
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
